modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/constants/alertActivationKeysDownloadText.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+export const ALERT_ACTIVATION_KEYS_DOWNLOAD_TEXT = {
+	danger: 'Unable to export keys, please try again.',
+	success: 'Activation key details successfully exported.',
```

Comment:
```suggestion
	[ALERT_DOWNLOAD_TYPE.danger]: 'Unable to export keys, please try again.',
	[ALERT_DOWNLOAD_TYPE.success]: 'Activation key details successfully exported.',
```
We can sync those 2 objects that you've created and I think will work well, what do you think?